doc/sphinx/Pacemaker_Explained/alerts.rst

```diff
@@ -1,422 +1,496 @@
+                <recipient id="my-alert-recipient" value="some-address"/>
+            </alert>
+         </alerts>
+      </configuration>
+   
+In the above example, the cluster will call ``my-script.sh`` for each event,
+passing the recipient ``some-address`` as an environment variable.
+
+The recipient may be anything the alert agent can recognize -- an IP address,
+an e-mail address, a file name, whatever the particular agent supports.
    
-   Meta-attributes can be configured per alert agent and/or per recipient.
    
-   .Alert configuration with meta-attributes
-   =====
-   [source,XML]
-   -----
-   <configuration>
-       <alerts>
-           <alert id="my-alert" path="/path/to/my-script.sh">
+.. index::
+   single: alert; meta-attributes
+   single: meta-attribute; alert meta-attributes
+
+Alert Meta-Attributes
+#####################
+   
+As with resource agents, meta-attributes can be configured for alert agents
+to affect how Pacemaker calls them.
+   
+.. table:: **Meta-Attributes of an Alert**
+   
+   +------------------+---------------+-----------------------------------------------------+
+   | Meta-Attribute   | Default       | Description                                         |
+   +==================+===============+=====================================================+
+   | timestamp-format | %H:%M:%S.%06N | Format the cluster will use when sending the        |
+   |                  |               | event's timestamp to the agent. This is a string as |
+   |                  |               | used with the ``date(1)`` command.                  |
+   |                  |               |                                                     |
+   |                  |               | .. index::                                          |
+   |                  |               |    single: alert; meta-attribute, timestamp-format  |
+   |                  |               |    single: meta-attribute; timestamp-format (alert) |
+   |                  |               |    single: timestamp-format; alert meta-attribute   |
+   +------------------+---------------+-----------------------------------------------------+
+   | timeout          | 30s           | If the alert agent does not complete within this    |
+   |                  |               | amount of time, it will be terminated.              |
+   |                  |               |                                                     |
+   |                  |               | .. index::                                          |
+   |                  |               |    single: alert; meta-attribute, timeout           |
+   |                  |               |    single: meta-attribute; timeout (alert)          |
+   |                  |               |    single: timeout; alert meta-attribute            |
+   +------------------+---------------+-----------------------------------------------------+
+   
```

Comment:
Index entries should go before table text.

Reply:
whoops, missed those ... done